src/SprykerEco/Zed/Unzer/Business/Oms/Command/ChargeOmsCommand.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\Unzer\Business\Oms\Command;
+
+use Generated\Shared\Transfer\OrderTransfer;
+use SprykerEco\Zed\Unzer\Business\Payment\ProcessorResolver\UnzerPaymentProcessorStrategyResolverInterface;
+
+class ChargeOmsCommand extends AbstractOmsCommand implements UnzerOmsCommandByOrderInterface
+{
+    /**
+     * @var \SprykerEco\Zed\Unzer\Business\Payment\ProcessorResolver\UnzerPaymentProcessorStrategyResolverInterface
+     */
+    protected $paymentProcessorStrategyResolver;
+
+    /**
+     * @param \SprykerEco\Zed\Unzer\Business\Payment\ProcessorResolver\UnzerPaymentProcessorStrategyResolverInterface $paymentProcessorStrategyResolver
+     */
+    public function __construct(
+        UnzerPaymentProcessorStrategyResolverInterface $paymentProcessorStrategyResolver
+    ) {
+        $this->paymentProcessorStrategyResolver = $paymentProcessorStrategyResolver;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\OrderTransfer $orderTransfer
+     * @param int[] $salesOrderItemIds
```

Comment:
```suggestion
     * @param array<int> $salesOrderItemIds
```
same for other cases across PR